.github/workflows/static_analysis.yml

```diff
@@ -0,0 +1,81 @@
+name: Static Analysis
```

Comment:
Random idea:
Put commit naming/ clang-format and clang-tidy in one workflow?

Reply:
What are the benefits? Less time needed for CI?

Reply:
Just 
```
?????/Commit Naming
?????/Clang Format
?????/Clang Tidy
```

Just grouped together.

Reply:
I don't know what to say about this. I leave this up to you guys.

Reply:
We can decide about this in the following ticket: https://github.com/inexorgame/vulkan-renderer/issues/380